packages/govern-contract-utils/contracts/address-utils/AddressUtils.sol

```diff
@@ -12,4 +12,10 @@ library AddressUtils {
     function toAddress(address addr) internal pure returns (address payable) {
         return address(bytes20(addr));
     }
+
+    function isContract(address addr) internal view returns (bool result) {
+        assembly {
+            result := extcodesize(addr)
```

Comment:
Jus to be safe, we might as well explicitly cast this to a bool. It's a small gas overhead, but feels better than leaving around the original uint256 size in the stack.

Reply:
Because it is anyways just in the stack of that call do I think is it totally okay to not convert it to a boolean and to save this gas.